src/utils/MainApi.js

```diff
@@ -0,0 +1,55 @@
+class MainApi {
+    constructor(baseUrl) {
+        this._baseUrl = baseUrl;
+    }
+
+    _checkResponse(res) {
+        if (res.ok) {
+            return res.json();
+        }
+        return Promise.reject(`Error ${res.status}`);
+    }
+
+    getUserInfo() {
+        return fetch(`${this._baseUrl}/users/me`, {
+            headers: {
+                Authorization: `Bearer ${localStorage.getItem("jwt")}`,
+                "Content-Type": "application/json",
+            },
+        });
```

Comment:
Please, use `_checkResponse` in every request inside `MainApi`. It's responsibility of this class

Reply:
Done